Report.cs

```diff
@@ -0,0 +1,68 @@
+using System.Net.Sockets;
+
+namespace PetrolStation;
+using  System.Collections.Generic;
+
+public class Report
+{
+   private List<Receipt> _receiptList = new List<Receipt>();
+   public void Add(Receipt toAdd)

```

Comment:
Tak naprawdę, to klasa Report powinna mieć tylko metodę Generate() a wszystkie rachunki pobierać z jednego source of truth